frontend/fingertips-frontend/components/molecules/BenchmarkSelectArea/index.tsx

```diff
@@ -0,0 +1,57 @@
+import {
+  SearchParamKeys,
+  SearchStateManager,
+  SearchStateParams,
+} from '@/lib/searchStateManager';
+import { useLoadingState } from '@/context/LoaderContext';
+import { usePathname, useRouter } from 'next/navigation';
+import { StyledFilterSelect } from '@/components/styles/StyledFilterSelect';
+import { AreaWithoutAreaType } from '@/lib/common-types';
+
+interface BenchmarkSelectAreaProps {
+  availableAreas: AreaWithoutAreaType[];
+  benchmarkAreaSelectedKey: SearchParamKeys;
+  searchState: SearchStateParams;
+}
+
+export function BenchmarkSelectArea({
+  availableAreas,
+  benchmarkAreaSelectedKey,
+  searchState,
+}: Readonly<BenchmarkSelectAreaProps>) {
+  const pathname = usePathname();
+  const { replace } = useRouter();
+  const { setIsLoading } = useLoadingState();
+
+  const searchStateManager = SearchStateManager.initialise(searchState);
+
+  const benchmarkAreaSelected = (valueSelected: string) => {
```

Comment:
```suggestion
  const onBenchmarkAreaSelected = (valueSelected: string) => {
```
This is very much an opinion thing, so feel free to ignore, but I do like the onSomething event handlers to start with `on` as I find it instantly recognisable as being an event handler